.circleci/config.yml

```diff
@@ -42,23 +41,59 @@ jobs:
             - node_modules
           key: v2-dependencies-npm-{{ checksum "package-lock.json" }}
       - run: npm run uswds-build
-      # We export the file list in pa11y_targets so that we can make pa11y-ci scan only those changed files in the "Run pa11yci" step
       - run:
-          name: Jekyll build
+          name: Jekyll build and store pa11y_targets
+          # We export the changed file list in pa11y_targets source
+          # that we can make pa11y-ci scan only those changed files
+          # in the "Run pa11y scan on changed files" step if it's
+          # needed, a full site scan does not use the pa11y_targets file
           command: |
             bundle exec jekyll build
-            echo "export PA11Y_TARGETS=$(cat pa11y_targets | base64 --wrap=0)" >> "$BASH_ENV"
-            source "$BASH_ENV"
+            if test -f pa11y_targets; then
```

Comment:
todo: I need to add an else to this check, if no files were changed that would cause the pa11y_targets file to be written to the file system we should still push some value into the environment variable that we can apply conditional logic to for the Pa11y scan step which could be skipped all together. I'm thinking of a change to a file(s) that wouldn't create the pa11y_targets file is things like `.eleventy.js` or dependabot updating `package.json`.

Reply:
After some more review and testing, this doesn't need an else, I mistakenly had the "Do a pa11y scan on the changed files" logic placed inside the `if [[ -z "${PA11Y_TARGETS}" ]]; then` check when that if statement is actually checking for a zero length result which happens when the env var is empty or unset.